extensions/gitpod/package.json

```diff
@@ -29,8 +29,39 @@
 	],
 	"activationEvents": [
 		"*",
-		"onCommand:gitpod.api.autoTunnel"
+		"onCommand:gitpod.api.autoTunnel",
+		"onAuthenticationRequest:gitpod"
 	],
+	"contributes": {
+		"authentication": [
+			{
+				"label": "Gitpod",
+				"id": "gitpod"
+			}
+		],
+		"configuration": {
+			"title": "Settings sync",
+			"properties": {
+				"configurationSync.store": {
+					"type": "object"
+				}
+			}
+		},
+		"commands": [
+			{
+				"command": "gitpod.auth.remove",
+				"category": "Gitpod",
+				"enablement": "gitpod.addedSyncProvider",
+				"title": "Disable Gitpod Settings Sync Provider"
```

Comment:
Is it surfaced somewhere on UI? It sounds very technical, but I could not find it on UI. can we remove `title` property to avoid confusion?

Reply:
it's in the command pallete

Reply:
I thought it is `Gitpod: Turn Setting Sync On/Off`?

Reply:
I changed it because it's misleading, it doesn't turn settings sync at all, for that user has to do it manually by invoking `Settings Sync: Turn On...` command or in the menu option

Reply:
I don't think anyone will understand what `Gitpod Settings Sync Provider` means either.

Can we contribute following items instead:
- `Settings Sync: Enable signing in with Gitpod account`
- `Settings Sync: Disable signing in with Gitpod account`

And fix error messaging correspondingly to something `Failed to enable singing in with Gitpod account: ${e}` or `Failed to disable singing in with Gitpod account: ${e}`

Reply:
I like @akosyakov's idea, but I am not sure if we can have the `Settings Sync:` prefix, since the extension is named `Gitpod` and so all commands start with that. This means that the command would be `Gitpod: Settings Sync: Enable signing in with Gitpod account`.
To be honest I am not sure what to change this to, but perhaps something like `Gitpod: Sign In to VS Code`? It is quite difficult to make this text short and concise, it is important that we tell the user that:
a) this is only useful for settings sync
b) this does not explicitly enable it, but it is the first step

Reply:
You can drop category and just provide title: `Settings Sync: Enable signing in with Gitpod account`.

Reply:
I think we can make it shorter and drop the **account** there (I believe it's not necessary), but otherwise that sounds good 👍

Reply:
done